nano/node/blockprocessor.cpp

```diff
@@ -47,6 +48,32 @@ nano::block_processor::block_processor (nano::node & node_a, nano::write_databas
 			block_processed.notify (result, context);
 		}
 	});
+
+	queue.max_size_query = [this] (auto const & origin) {
+		switch (origin.source)
+		{
+			case nano::block_source::live:
+				return config.max_peer_queue;
+			default:
+				return config.max_system_queue;
+		}
+	};
+
+	queue.priority_query = [this] (auto const & origin) -> size_t {
+		switch (origin.source)
+		{
+			case nano::block_source::live:
+				return config.priority_live;
+			case nano::block_source::bootstrap:
+			case nano::block_source::bootstrap_legacy:
+			case nano::block_source::unchecked:
+				return config.priority_bootstrap;
+			case nano::block_source::local:
+				return config.priority_local;
+			default:
+				return 1;
```

Comment:
This will give block_source::forced a priority of 1.
I would expect forced to have a higher priority.

Reply:
Okay, I can make it higher priority, but can you explain exactly why it should be higher priority?